web_project/settings.py

```diff
@@ -84,7 +84,12 @@
 ]
 
 WSGI_APPLICATION = 'web_project.wsgi.application'
-
+ASGI_APPLICATION = 'web_project.asgi.application'
+CHANNEL_LAYERS = {
+    "default": {
+        "BACKEND": "channels.layers.InMemoryChannelLayer" # for dev use only!!
```

Comment:
should be modified to account for the DEBUG environment variable, & probably will use Redis for the Channel Layer in Prod. That will require another dependency, I think